EDAM_dev.owl

```diff
@@ -37319,6 +37319,32 @@ experiments employing a combination of technologies.</oboInOwl:hasDefinition>
 
 
 
+    <!-- http://edamontology.org/format_4019 -->
+
+    <owl:Class rdf:about="http://edamontology.org/format_4019">
+        <created_in>1.26</created_in>
+        <rdfs:label>HOC</rdfs:label>
+        <oboInOwl:hasExactSynonym>High Order Calculator</oboInOwl:hasExactSynonym>
+        <file_extension>hoc</file_extension>
+        <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#edam"/>
+        <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#formats"/>
+        <oboInOwl:hasDefinition>hoc (High Order Calculator) is an interpreted programming language used as the primary scripting language for the NEURON simulator.</oboInOwl:hasDefinition>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_2330"/>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_2013"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+            	<owl:onProperty rdf:resource="http://edamontology.org/is_format_of"/>
+                <owl:someValuesFrom rdf:resource="http://edamontology.org/data_3241"/>
```

Comment:
` property_value = term['owl:someValuesFrom']['@id']`
`TypeError: list indices must be integers or slices, not str`
`Error: Process completed with exit code 1.`


this's the reason for failing test cases @matuskalas 

can you give an example using these two lines, for how should I write them for type consistency.
I wrote them on two lines, but I guess this might work on the same line since data type is list here, according to the failing test cases.